### PR TITLE
3. Create scheduleInterview function

### LINKED ISSUE #4: Create a function to Schedule Interview with name scheduleInterview, that accepts (ID:string, ScheduleDate:string)
- The status of student is set to "Under Interview."
- InterviewDate property of student is set.

### PR DESCRIPTION
**Pull request by Abdulwalie02**
- The function for scheduleInterview is created.
- Value of scheduleDate variable is pre-determined as an example.
- Date-type variable is used.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -14,11 +14,24 @@ var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probat
...
-    return await db.get(id, function(err, value){
+    await db.get(id, function(err, value){
         if(err){
             console.log(err);
         } else{
             console.log(value, status[0]);
         }
     })
+    var scheduleDate = new Date('March 1, 2021 08:30:00');
+    return await scheduleInterview(id, scheduleDate);
```
Comment (fencim) [inline main.js]: removing this lines, move it after line 11

File: main.js [hunk 2]
```diff
@@ -14,11 +14,24 @@ var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probat
...
+
+async function scheduleInterview(id, scheduleDate){
+    return await db.get(id, function(err, value){
+        if(err){
+            console.log(err);
+        } else{
+            console.log(value, status[1]);
+            console.log('Interview date on', scheduleDate);
+        }
+    })
```
Comment (fencim) [inline main.js]: use await to get the student object.
```suggestion
    var student = await db.get(id);
    student.interviewDate = scheduleDate;
    await db.put(id, student);
    console.log('Interview date on', scheduleDate);
```

**Review by fencim — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (Abdulwalie02) [reply main.js]: the lines are moved sir but we have to return the value of id variable in the acceptStudent function.

File: main.js (hunk 2 above)
Comment (Abdulwalie02) [reply main.js]: it is updated sir, but with few changes for the console.log().

**Review by fencim — APPROVED** [review]

**Pull request merged**